以Memory作为后端起服务时，使用Client删除元数据报错 Undefined vertex label: '~task'

### Expected behavior 期望表现
成功删除元数据

### Actual behavior 实际表现
报错

```
Exception in thread "main" class com.baidu.hugegraph.HugeException: Failed to update/query TaskStore
	at com.baidu.hugegraph.exception.ServerException.fromResponse(ServerException.java:44)
	at com.baidu.hugegraph.client.RestClient.checkStatus(RestClient.java:63)
	at com.baidu.hugegraph.rest.RestClient.delete(RestClient.java:198)
	at com.baidu.hugegraph.api.schema.EdgeLabelAPI.delete(EdgeLabelAPI.java:73)
	at com.baidu.hugegraph.driver.SchemaManager.removeEdgeLabel(SchemaManager.java:135)
	at com.baidu.hugegraph.example.SingleExample.lambda$main$4(SingleExample.java:181)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at com.baidu.hugegraph.example.SingleExample.main(SingleExample.java:180)
Caused by: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Undefined vertex label: '~task'

```

### Steps to reproduce the problem 复现步骤
1. 以Memory作为后端启动服务
2. 使用HugeClient添加一些数据，比如 SingleExample 的数据
3. 删除元数据，报错

--- a/hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/memory/InMemoryDBStore.java
+++ b/hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/memory/InMemoryDBStore.java
@@ -154,6 +154,11 @@ public class InMemoryDBStore implements BackendStore {
     @Override
     public void open(HugeConfig config) {
         LOG.debug("open()");
+        /*
+         * Memory store need to init some system property,
+         * like task related propertykeys and vertexlabels
+         */
+        this.provider.init();
     }
 
     @Override

--- a/hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTask.java
+++ b/hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTask.java
@@ -320,7 +320,7 @@ public class HugeTask<V> extends FutureTask<V> {
         list.add(this.create);
 
         list.add(P.RETRIES);
-        list.add( this.retries);
+        list.add(this.retries);
 
         if (this.description != null) {
             list.add(P.DESCRIPTION);

